.github/workflows/linux.yml

```diff
@@ -867,7 +867,7 @@ jobs:
     defaults:
       run:
         shell: bash
-    runs-on: ${{ github.event_name == 'schedule' && 'ubuntu-20.04-8-cores' || 'ubuntu-20.04'}}
+    runs-on: ${{ github.event_name == 'schedule' && 'aks-linux-16-cores' || 'aks-linux-16-cores'}}
```

Comment:
revert runner for `pre-commit`.
Now it just makes sure that self-hosted runner works

Reply:
I think we should use the same logic for self-hosted runners ( aks-linux-4-cores for small jobs and aks-linux-8-cores/aks-linux-16-cores for huge jobs) to reduce billing

Reply:
Fixed.